apps/auth/database.py

```
@@ -0,0 +1,16 @@
+from pymongo import MongoClient
+from pymongo.errors import ConnectionFailure
+from config import MONGO_URI, REDIS_HOST, REDIS_PORT
```

Ruff (0.8.2)

`config.REDIS_HOST` imported but unused

Remove unused import

(F401)